SecondStage/src/main/java/errorandexceptions/Faculty.java

```diff
@@ -0,0 +1,64 @@
+package errorandexceptions;
```

Comment:
As there's no pom file the project is not imported properly. The package structure is incorrect. Also note that package names should be in lower case (SecondStage is also a package)